netsim/ansible/templates/bgp/junos.j2

```diff
@@ -79,24 +79,32 @@ protocols {
 {%   for n in bgp.neighbors if n[af] is defined and n.type == 'ibgp' %}
       neighbor {{ n[af] }} {
         description {{ n.name }};
+{%     if n.activate[af]|default(true) %}
         family {{ 'inet' if af == 'ipv4' else 'inet6' }} {
           unicast;
         }
+{%     endif %}
       }
 {%   endfor %}
     }
 {% endfor %}
     group ebgp-peers {
       export ebgp-export;
       advertise-inactive;
-{% for n in bgp.neighbors if n.type == 'ebgp' %}
+{# treat localas_ibgp as ebgp session --> no usage of local address #}
+{% for n in bgp.neighbors if (n.type == 'ebgp' or n.type == 'localas_ibgp') %}
```

Comment:
Perhaps ```localas_ibgp ``` is similar to ```ebgp``` in that it does not use a local address, but similar to ```ibgp``` in that the ```ibgp-export``` policy should apply?

Reply:
I didn't want to add overcomplexity with the need to declare a new group or too much per-neighbor options.

If we find use cases where this is not working we can either fix it or declare it as current limitation or caveat.

My 2c